jest.config.js

```diff
@@ -16,4 +16,8 @@ module.exports = {
       "<rootDir>/src/__mocks__/fileMock.js",
     "\\.(css|less)$": "<rootDir>/src/__mocks__/styleMock.js",
   },
+  transformIgnorePatterns: ["node_modules/(?!(serialize-error)/)"],
```

Comment:
`serialize-error` is an ESM package. This is needed to ensure Jest doesn't throw up on us.